rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  def initialize(data, book, person)
+    @data = data
+    @person = person
+    @book = book
+    person.rentals << self
+    book.rentals << self
+  end
+
+  attr_accessor :date, :book, :person
```

Comment:
- [ ]  Please follow the first comment I made above, and do the same thing: Move `attr_accessor :date, :book, :person` below `class Rental`. This also helps you to follow Ruby best style

Reply:
Thanks @OLIPLICHE for your feedback